Godeps/_workspace/src/github.com/kr/pty/doc.go

```diff
@@ -0,0 +1,16 @@
+// Package pty provides functions for working with Unix terminals.
```

Comment:
Good question. I'll have to look into it. Do you know what terminal app or apps we need to try to support on Windows?